CoreLogic.php

```diff
@@ -1156,7 +1156,7 @@ function HasKeyword($cardID, $keyword, $player="", $index=-1){
     case "Raid": return RaidAmount($cardID, $player, $index, true) > 0;
     case "Grit": return HasGrit($cardID, $player, $index);
     case "Restore": return RestoreAmount($cardID, $player, $index) > 0;
-    case "Bounty": return CollectBounty($player, $cardID, "-", "-", true) > 0;
+    case "Bounty": return CollectBounty($player, $cardID, $cardID, false, $player, true) > 0; // Since we don't have information about "exhausted" and "owner," this data may be imprecise in very rare cases.
```

Comment:
bounty owner is irrelevant in report mode (with current cards).
and all Exhaust-Conditional, Bountied cards have other keywords anyway (Synara San has Grit and Unlicensed Headhunter has Saboteur).
of course this can change with new sets, but we should be good for now.

thanks for fixing this!